atlasdb-jepsen-tests/src/test/java/com/palantir/atlasdb/jepsen/events/EventTest.java

```diff
@@ -34,6 +37,9 @@
     public void makeSureWeCanHaveNullValues() {
         Map<Keyword, Object> keywordMap = new HashMap<>();
         keywordMap.put(Keyword.intern("type"), Keyword.intern("info"));
+        keywordMap.put(Keyword.intern("f"), Keyword.intern(JepsenConstants.START_FUNCTION));
+        keywordMap.put(Keyword.intern("process"), Keyword.intern(JepsenConstants.NEMESIS_PROCESS));
+        keywordMap.put(Keyword.intern("time"), SOME_TIME);
```

Comment:
I've just noticed that we're not actually checking the values of the constructed InfoEvent here! I think checking that you get an empty Optional, as opposed to just a null value, would be useful.